src/main/kotlin/com/ampnet/auditornode/controller/websocket/AuditWebSocket.kt

```diff
@@ -138,7 +145,7 @@ class AuditWebSocket @Inject constructor(
             is RpcError -> webSocketApi.sendInfoMessage(RpcErrorInfoMessage)
         }
 
-        session.close(CloseReason.ABNORMAL_CLOSURE)
+        session.close(CloseReason.NORMAL)
```

Comment:
I kind of misused close reason here - it is not like HTTP status codes. Abnormal closure means that the client can potentially reconnect and continue communication. Micronaut was issuing a warning when I was using the abnormal closure code. So I set everything to normal because that is the close reason we should use when communication is finished successfully. Sending an error message is still considered a successful as it does not mean any error in the web socket protocol itself.